cmd/install.go

```diff
@@ -79,6 +68,37 @@ var installCommand = &cobra.Command{
 			pkg.Entrypoint = []string{customEntrypoint}
 		}
 
+		installPath := viper.GetString("install_path")
+		pm := packages.NewPackageManager(installPath)
+
+		var installed *packages.Package
+		hasInstall := pm.HasInstallation(pkg.Name)
+		if hasInstall {
+			installed, err = pm.Load(pkg.Name)
+			if !forceInstall && err != nil {
+				return fmt.Errorf("there's already an installation of %s, but there was an error loading the package, err: %s", pkg.Name, err.Error())
+			}
+
+			fmt.Printf("Looks like you already have %s installed as %s.\n", installed.Image, path.Join(installPath, pkg.Name))
+
+			if !assumeYes {
+				if equal, err := installed.HasChanges(ctx, cli); err != nil {
+					return err
+				} else if !equal {
+					if !prompter.YN("There are differences between the installed version of the package and the image.\nAre you sure you would like to overwrite these changes?", false) {
+						return fmt.Errorf("Not installing package")
+					}
+				} else if pkg.Image == installed.Image {
+					return fmt.Errorf("%s would generate the same package, nothing to do\n", pkg.Image)
```

Comment:
I am foreseeing that this will make the main command to exit with a status != 0. I would tend to think that the exit code should be 0 in case the package is already installed with the same values, the current state matches expectations

Reply:
Yep that's true i'll change this to write a message and exit without error.